linetools/analysis/voigt.py

```diff
@@ -24,28 +21,29 @@
 
 #from xastropy.xutils import xdebug as xdb
 
-spl = 29979245800.0 # cm/s
-# 
+spl = 29979245800.0  # cm/s
```

Comment:
Should we come up with a naming convention for the speed of light? I suggest using C_cms or C_kms for global variables...